dev-libs/libvterm/libvterm-0.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit flag-o-matic
+
+DESCRIPTION="An abstract library implementation of a VT220/xterm/ECMA-48 terminal emulator"
+HOMEPAGE="http://www.leonerd.org.uk/code/libvterm/"
+SRC_URI="https://launchpad.net/${PN}/trunk/v${PV}/+download/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-lang/perl
+	sys-devel/libtool
+	virtual/pkgconfig
+"
+DEPEND=""
+RDEPEND="!dev-libs/libvterm-neovim"
+
+src_prepare() {
+	default
+	if use test; then
+		# Remove broken test
+		rm "t/63screen_resize.test" || die
```

Comment:
Remove it unconditionally. This isn't going to harm anyone who's not running tests, is it?

Reply:
I don't think so, good idea